Automattic/woocommerce-payments
# Moves all unit test files to single folder under '/tests'

Fixes #759 

#### Changes proposed in this Pull Request

* Moves all unit test files into a folder under `/test` called `/unit`.
* Updates to the `phpunit` config file to correctly find these tests.

#### Testing instructions

* Run the command `npm run test:php` from the project root and verify that the tests run and that all tests are successful.

-------------------
